.gitpod.yml

```diff
@@ -2,19 +2,19 @@ image:
   file: .gitpod.Dockerfile
 
 tasks:
-  - name: Restart Kurtosis Engine
+  - name: (1) Boot Kurtosis Engine
     command: |
       kurtosis engine restart
       clear
-  - name: Run Fake Package
+  - name: (2) Initialize workspace
```

Comment:
Run fake package is for us to get analytics about the quickstart gitpod being run, but I'd obfuscate that to be "initialize"